lms/djangoapps/certificates/api.py

```diff
@@ -34,6 +34,15 @@
 log = logging.getLogger("edx.certificate")
 
 
+def is_passing_status(cert_status):
```

Comment:
I see. What's wrong with importing models across packages? I've heard this given as a justification for api.py before, but haven't questioned it.

If we're going to stick with api.py, shouldn't this function be used in [lms/djangoapps/instructor_task/tasks_helper.py](https://github.com/edx/edx-platform/pull/11341/files#diff-9b7fcb710c553d15f0d138a19ec082d0R1451)?